qds_sdk/cloud/cloud.py

```diff
@@ -0,0 +1,22 @@
+from qds_sdk.qubole import Qubole
+
+class Cloud:
+
+    @staticmethod
+    def get_cloud():
+        cloud = Qubole.cloud
+        if cloud == "azure":
+            import qds_sdk.cloud.azure_cloud
+            return qds_sdk.cloud.azure_cloud.AzureCloud()
+        elif cloud == "oracle_bmc":
+            import qds_sdk.cloud.oracle_bmc_cloud
+            return qds_sdk.cloud.oracle_bmc_cloud.OracleBmcCloud()
+        else:
+            import qds_sdk.cloud.aws_cloud
+            return qds_sdk.cloud.aws_cloud.AwsCloud()
+
+    def create_parser(self):
```

Comment:
Function signature is incorrect. In aws_cloud.py, azure_cloud.py etc the signature id `def create_parser(self, argparser)`.